gradle/libs.versions.toml

```diff
@@ -14,6 +15,9 @@ kotlinx-serialization = "org.jetbrains.kotlinx:kotlinx-serialization-json:1.6.0"
 
 maven-modelBuilder = "org.apache.maven:maven-model-builder:3.9.4"
 
+# Dummy to get renovate updates, the version is used in rootProject build.gradle with spotless.
+ktlint-dummy = { module = "com.pinterest.ktlint:ktlint-cli", version.ref = "ktlint" }
```

Comment:
Please inline the version like this: https://github.com/cashapp/molecule/blob/trunk/gradle/libs.versions.toml#L44

Then you can access the version from the root like this: https://github.com/cashapp/molecule/blob/trunk/build.gradle#L79

Reply:
Done.